website/client/src/components/userMenu/profileStats.vue

```diff
@@ -266,7 +266,7 @@
           class="col-12 col-md-3"
         >
           <div class="box white row col-12">
-            <div class="col-9">
+            <div class="col-9 stat-points">
```

Comment:
could we use the `text-nowrap` class that comes with bootstrap? this way we won't have to create a new one every time